### PR TITLE
I139: Add 'Supported images' Default File Filter to Fusion UI

### PR DESCRIPTION
**Pull request by tst-ppenev**
* Added a 'Supported images' file filter to the _Asset Manager -> Imagery Resource -> Add_ dialog.

### CODE REVIEW

File: earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp [hunk 1]
```diff
@@ -226,7 +240,7 @@ QFileDialog* RasterAssetWidget::FileDialog() {
     file_dialog_->addFilter("Tiff/GeoTiff ( *.tif *.TIF )");
```
Comment (bradh) [inline earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp]: probably would be better as `TIFF/GeoTIFF`.

File: earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp [hunk 2]
```diff
@@ -226,7 +240,7 @@ QFileDialog* RasterAssetWidget::FileDialog() {
...
 
     // make the first filter current, which is the default "All Files (*)"
```
Comment (bradh) [inline earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp]: Is this comment still valid?

Comment (tst-ppenev) [inline earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp]: :) No, it's not. Thanks!

File: earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp (hunk 1 above)
Comment (tst-ppenev) [inline earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp]: Yeah, that's true.

**Comment by tst-ppenev** [comment]
@bradh: No, those commits were from a previous test. I got rid of them. Thanks.

**Comment by tst-ccamp** [comment]
@tst-ppenev This looks good, but while we're at it, I think it would make sense to also add this to the vector file dialog. Would that require significant effort?

**Comment by tst-ppenev** [comment]
@tst-ccamp: No, in fact, I may have already done that. I'm trying to a 'supported files' filter to the imagery, vector, and terrain dialogs.

**Comment by tst-ccamp** [comment]
@tst-ppenev I just tested it and did not see a "Supported Files" filter in vector like I see in imagery and terrain. Maybe there's a file that didn't get checked in?

**Comment by tst-ppenev** [comment]
@tst-ccamp: Yeah, I haven't pushed the commits, sorry.

File: earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp [hunk 3]
```diff
@@ -216,17 +219,29 @@ QFileDialog* RasterAssetWidget::FileDialog() {
         }
       }
     }
-    // For JPEG2000 and MrSID, check if GDAL is built
-    // with specific format support.
+    // Add file filters:
+    QString supported_files_filter = "Supported images (";
+    if (add_dem_file_filter)
```
Comment (asmitaw) [inline earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp]: Its confusing to have DEM for Imagery. We should not check for it and add it here. Its for Terrain only.

**Comment by tst-ppenev** [comment]
This has 3 approvals. I'm going to merge.

**Pull request merged**